risc0/zkvm/README.md

```diff
@@ -27,3 +27,23 @@ channel](https://www.youtube.com/@risczero) has many more videos as well.
   see if there's a workaround, and if you're using a workaround, you can track
   when it gets resolved to a permanent solution.
 * And more on [the RISC Zero website](https://www.risczero.com/)!
+
+# Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
+
+* cuda
+  * Optional (default is CPU only)
+  * Turns on GPU acceleration for guest code with CUDA
+* metal
+  * Optional (default is CPU only)
+  * Turns on GPU acceleration for guest code with Metal
+* profiler
+  * Optional, experimental
```

Comment:
I don't think this is experimental anymore

Reply:
It's optional mostly because it introduces a lot of dependencies (like protobuf) that are pretty heavy and aren't needed most of the time

Reply:
Ok I'll pull this out of the code comments as well then.